src/Cmgmyr/Messenger/examples/MessagesController.php

```diff
@@ -30,7 +30,9 @@ public function index()
         // All threads that user is participating in, with new messages
         // $threads = Thread::forUserWithNewMessages(Auth::id())->latest('updated_at')->get();
 
-        return view('messenger.index', compact('threads'));
+        return response()->view('portal.messenger.index', [
+            "threads" => $threads,
```

Comment:
There is also a reason more important than just style - **performance**!
Strings in double-quotes are tried to parse `"Hello {$name}"` for example - in single-quotes not. That's why `'Hello '.$name` is faster than `"Hello {$name}"`.